packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/http_clients/api_http_client.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:dio/adapter.dart';
+import 'package:dio/dio.dart';
+{{#analytics}}
+import '../interceptors/analytics_interceptor.dart';{{/analytics}}
+import '../interceptors/auth_interceptor.dart';
+import '../interceptors/log_interceptor.dart';
+
+/// An HTTP client for use with the main backed API.
+class ApiHttpClient with DioMixin implements Dio {
+  ApiHttpClient() {
```

Comment:
Don't you think it will be useful to have the `options` as an argument so that its value can be changed if needed ?